{"status":"OK","subtests":[{"name":"# AUDIT TASK RUNNER STARTED.","status":"PASS"},{"name":"> [ref-distance-error] ","status":"PASS"},{"name":"X new PannerNode(c, {refDistance: -1}) threw \"TypeError\" instead of function RangeError() { [native code] }.","status":"FAIL"},{"name":"X new PannerNode(c, {refDistance: 0}) incorrectly threw TypeError: \"Function expected\".","status":"FAIL"},{"name":"X new PannerNode(c, {refDistance: 5e-324}) incorrectly threw TypeError: \"Function expected\".","status":"FAIL"},{"name":"Executing \"ref-distance-error\"","status":"FAIL"},{"name":"< [ref-distance-error] 3 out of 3 assertions were failed.","status":"FAIL"},{"name":"> [max-distance-error] ","status":"PASS"},{"name":"X new PannerNode(c, {maxDistance: -1}) threw \"TypeError\" instead of function RangeError() { [native code] }.","status":"FAIL"},{"name":"X new PannerNode(c, {maxDistance: 0}) threw \"TypeError\" instead of function RangeError() { [native code] }.","status":"FAIL"},{"name":"X new PannerNode(c, {maxDistance: 5e-324}) incorrectly threw TypeError: \"Function expected\".","status":"FAIL"},{"name":"Executing \"max-distance-error\"","status":"FAIL"},{"name":"< [max-distance-error] 3 out of 3 assertions were failed.","status":"FAIL"},{"name":"> [min-distance] ","status":"PASS"},{"name":"Executing \"min-distance\"","status":"FAIL"},{"name":"< [min-distance] All assertions passed. (total 0 assertions)","status":"PASS"},{"name":"> [max-distance] ","status":"PASS"},{"name":"Executing \"max-distance\"","status":"FAIL"},{"name":"< [max-distance] All assertions passed. (total 0 assertions)","status":"PASS"},{"name":"# AUDIT TASK RUNNER FINISHED: 2 out of 4 tasks were failed.","status":"FAIL"}]}